crates/matrix-sdk-base/src/client.rs

```diff
@@ -541,8 +541,7 @@ impl BaseClient {
                 for (user_id, rooms) in e.content.iter() {
                     for room_id in rooms {
                         trace!(
-                            room_id = room_id.as_str(),
-                            target = user_id.as_str(),
+                            ?room_id, target = ?user_id,
```

Comment:
Here and in the rest of this patch (`Avoid verbose string conversions when logging / printing`), you're replacing `XYZ.as_str()` with `?XYZ`, which will use the `Debug` impl for the underlying `str` instead of the plain `Display`. I think we should use `%room_id` instead?

Reply:
No, we use `?` instead of `%` for ruma ID types because like the `Value` impl for `str`, the output is a quoted string (with `%`, it would *not* be quoted).